packages/frontend/src/common/context/ColorThemeContext.tsx

```diff
@@ -22,8 +30,53 @@ declare module "@mui/material/styles" {
     highlightColorPrimary: string;
     highlightColorSecondary: string;
   }
-}
 
+  interface TypographyVariants {
+    title: React.CSSProperties;
+    menuTitle: React.CSSProperties;
+  }
+
+  // allow configuration using `createTheme`
+  interface TypographyVariantsOptions {
+    title?: React.CSSProperties;
+    menuTitle: React.CSSProperties;
+  }
+}
+const components = {
+  components: {
+    MuiTypography: {
+      defaultProps: {
+        variantMapping: {
+          title: "h1",
+          menuTitle: "span",
+        },
+      },
+    },
+  },
+};
+const typography = {
```

Comment:
find type. Maybe extend TypographyVariantsOptions from existing typography variants?

Reply:
Were able to set it to a subtype of the VariantOptions